trillian/migrillian/core/util.go

```diff
@@ -59,27 +60,25 @@ func ValidateConfig(cfg *configpb.MigrationConfig) error {
 }
 
 func buildLogLeaf(logPrefix string, index int64, entry *ct.LeafEntry) (*trillian.LogLeaf, error) {
-	logEntry, err := ct.LogEntryFromLeaf(index, entry)
-	if x509.IsFatal(err) {
-		return nil, fmt.Errorf("failed to build LogEntry[%d]: %v", index, err)
+	rle, err := ct.RawLogEntryFromLeaf(entry, index)
```

Comment:
As it's only just gone in, can we also reverse the order of arguments in `RawLogEntryFromLeaf` so it's consistent with `LogEntryFromLeaf` please.

Reply:
Done.